app/assets/src/components/visualizations/TidyTree.js

```diff
@@ -24,7 +24,8 @@ export default class TidyTree {
         useCommonName: false,
         minNonCollapsableChildren: 2,
         smallerFont: 8,
-        largerFont: 12
+        largerFont: 12,
+        persistCollapsedInUrl: false
```

Comment:
This visualization should be independent of the view outside,
(There are few fixes need to be done to keep it independent).
Use callbacks to handle state changes instead.

Reply:
Also, the previous state should be passed in the data nodes...